packages/Organisms/Markup/package.json

```diff
@@ -10,9 +10,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@dpc-sdp/ripple-global": "1.0.0-alpha.43",
-    "@dpc-sdp/ripple-document-link": "1.0.0-alpha.43",
     "@dpc-sdp/ripple-button": "1.0.0-alpha.43",
+    "@dpc-sdp/ripple-document-link": "1.0.0-alpha.43",
```

Comment:
need to be `1.0.0-alpha.45`

Reply:
ah good catch, i wonder why that didn't get updated by lerna?

Reply:
@dylankelly Lerna will increment all package's current version to next version, but when you rebase/merge branch your working branch code is outdated, the version is not same as the current version, so it didn't get picked by Lerna. It happened before, so I did check it. 🤓